en/docs/install-and-setup/setup/api-controller/advanced-topics/configuring-different-endpoint-types.md

```diff
@@ -10,11 +10,11 @@ API Manager supports four (4) main types of endpoints as follows.
 4. AWS Lambda Endpoints
 
 !!! info
-    The following fields should be specified as your requirement when setting up different endpoints in `api_params.yaml` file.
+    The following fields should be specified as your requirement when setting up different endpoints in the params file.
```

Comment:
Any specific reason for changing this? Since this params file refer to the api_params.yaml, what is the need of changing it as such?